src/nonlinearshrinkage.jl

```diff
@@ -6,16 +6,17 @@ Analytical nonlinear shrinkage estimator. See docs for
 """
 struct AnalyticalNonlinearShrinkage{TEigen<:Union{Eigen,Nothing}} <: CovarianceEstimator
     decomp::TEigen
+    corrected::Bool
     function AnalyticalNonlinearShrinkage(decomp::TE=nothing) where TE<:Union{Eigen,Nothing}
         new{TE}(decomp)
```

Comment:
This function should have a keyword argument for `corrected`, just like the `Simple` estimator.